pkg/component/gardenercontrollermanager/configmaps.go

```diff
@@ -0,0 +1,168 @@
+				ConcurrentSyncs: pointer.Int(20),
+			},
+			Project: &controllermanagerv1alpha1.ProjectControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+				Quotas:          g.values.Quotas,
+			},
+			SecretBinding: &controllermanagerv1alpha1.SecretBindingControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+			},
+			Seed: &controllermanagerv1alpha1.SeedControllerConfiguration{
+				ConcurrentSyncs:    pointer.Int(20),
+				SyncPeriod:         &metav1.Duration{Duration: 10 * time.Second},
+				MonitorPeriod:      &metav1.Duration{Duration: 40 * time.Second},
+				ShootMonitorPeriod: &metav1.Duration{Duration: 300 * time.Second},
+			},
+			SeedExtensionsCheck: &controllermanagerv1alpha1.SeedExtensionsCheckControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      &metav1.Duration{Duration: 30 * time.Second},
+				ConditionThresholds: []controllermanagerv1alpha1.ConditionThreshold{{
+					Duration: metav1.Duration{Duration: 1 * time.Minute},
+					Type:     "ExtensionsReady",
+				}},
+			},
+			SeedBackupBucketsCheck: &controllermanagerv1alpha1.SeedBackupBucketsCheckControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      &metav1.Duration{Duration: 30 * time.Second},
+				ConditionThresholds: []controllermanagerv1alpha1.ConditionThreshold{{
+					Duration: metav1.Duration{Duration: 1 * time.Minute},
+					Type:     "BackupBucketsReady",
+				}},
+			},
+			Event: &controllermanagerv1alpha1.EventControllerConfiguration{
+				ConcurrentSyncs:   pointer.Int(10),
+				TTLNonShootEvents: &metav1.Duration{Duration: 2 * time.Hour},
+			},
+			ShootMaintenance: controllermanagerv1alpha1.ShootMaintenanceControllerConfiguration{
+				ConcurrentSyncs:                  pointer.Int(20),
+				EnableShootControlPlaneRestarter: pointer.Bool(true),
+			},
+			ShootQuota: &controllermanagerv1alpha1.ShootQuotaControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      &metav1.Duration{Duration: 60 * time.Minute},
+			},
+			ShootHibernation: controllermanagerv1alpha1.ShootHibernationControllerConfiguration{
+				ConcurrentSyncs:         pointer.Int(5),
+				TriggerDeadlineDuration: &metav1.Duration{Duration: 2 * time.Hour},
+			},
+			ShootReference: &controllermanagerv1alpha1.ShootReferenceControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+			},
+			ShootRetry: &controllermanagerv1alpha1.ShootRetryControllerConfiguration{
+				ConcurrentSyncs:   pointer.Int(5),
+				RetryPeriod:       &metav1.Duration{Duration: 10 * time.Minute},
+				RetryJitterPeriod: &metav1.Duration{Duration: 5 * time.Minute},
+			},
+			ManagedSeedSet: &controllermanagerv1alpha1.ManagedSeedSetControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      metav1.Duration{Duration: 30 * time.Minute},
+			},
+			ExposureClass: &controllermanagerv1alpha1.ExposureClassControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+			},
+		},
```

Comment:
Should we only specify the controllers when we overwrite default values? For example `ExposureClass` could be omitted and get defaulted by the config defaulting?

Reply:
Good point. Removed all values, which were defaults, so that the configuration is more concise.

Will be updated in the next commit.